ops/services/monitoring/_var.tf

```diff
@@ -30,7 +30,7 @@ variable "app_url" {
 
 variable "ai_ingest_cap_gb" {
   description = "Cap for data ingested into Application Insights, per day."
-  default     = 50
+  default     = 100
```

Comment:
that's a lot of data per day, Is that some cap we are currently reaching?

Reply:
I appreciate you calling this out. While not a huge deal, this would have needlessly increased data limits for lower environments. This value is set to 200 for production, but all other environments use this default.